src/components/Operations/Operation.jsx

```diff
@@ -13,15 +13,13 @@ import { useClient } from 'cozy-client'
 import React, { useCallback, useState } from 'react'
 
 import categories from 'assets/classes.json'
-import { capitalizeFirstLetter } from 'lib/utils'
+import { capitalizeFirstLetter, getBestCategory } from 'lib/utils'
 import OperationRemoveButton from './OperationRemoveButton'
 
 export const Operation = ({ operation }) => {
   const client = useClient()
 
-  const [category, setCategory] = useState(
-    operation.manualCategoryId || operation.cozyCategoryId || ''
-  )
+  const [category, setCategory] = useState(getBestCategory(operation))
```

Comment:
This is for the value of the default manualCategoryId and it just follows the banks guidelines

Reply:
Since it is for the manual category, the banks heuristic seem even less relevant: we should display the manual category is there is one, and nothing is there is none ; it does not make sense to display the cozyCategory in a "manual category" field